Detectors/ZDC/calib/src/BaselineCalibSpec.cxx

```diff
@@ -85,6 +89,13 @@ void BaselineCalibSpec::run(ProcessingContext& pc)
     mTimer.Reset();
     mTimer.Start(false);
   }
+  if (mRunStartTime == 0) {
+    mHistoFileMetaData = std::make_unique<o2::dataformats::FileMetaData>();
```

Comment:
Why do you create a metadata as a data member, moreover w/o even checking if it is needed at all? Instead you can create a local object instance in the sendOutput if it is needed